Portfolio/index.css

```diff
@@ -1 +1,64 @@
-/* Write your code below for PORTFOLIO */
+/* Write your code below for PORTFOLIO */
+*{
+      color: #515050;

```

Comment:
Consider using 'margin: 0;' instead of 'margin: 0%;'. The percentage symbol is not necessary for zero values.